src/archiefvernietigingscomponent/destruction/utils.py

```diff
@@ -0,0 +1,118 @@
+from datetime import datetime
```

Comment:
all the functions below are used to create a report. I think they deserve a special package or file with 'report' name